.gitignore

```diff
@@ -4,7 +4,8 @@ __pycache__/
 *.py[cod]
 
 # VSCode
-.vscode/
+.vscode/*
```

Comment:
maybe don't need the `*`?

Reply:
@rokroskar if you remove the `*` then simply the whole folder is gitignored and the line below `!.vscode/settings.json` does have any effects at all. So then the settings.json will not be checked in and it will be ignored.

Reply:
then does it need `.vscode/**/*` or similar to include subdirectories?